docs/pages/features/enhanced-session-recording.mdx

```diff
@@ -136,30 +147,29 @@ Linux ip-172-31-43-104.ec2.internal 4.19.72-25.58.amzn2.x86_64 x86_64 x86_64 x86
   </TabItem>
 </Tabs>
 
-## 2. Install BCC Tools
```

Comment:
This whole section is obsolete in 7.0 JFYI, the one #7341 is the up-to date for 7.0.

Reply:
Thanks, I'll merge in your changes (since it looks like that PR will go in first).

Reply:
Follow-up tested this further. Fewer changes than I thought after reading your comments - Will likely keep it as is with the current changes.